src/app/src/constants.js

```diff
@@ -73,3 +73,16 @@ export const msPerDay = msPerHour * 24;
 export const msPerWeek = msPerDay * 7;
 export const msPerMonth = msPerDay * 30; // roughly a month, 30 days
 export const msPerYear = msPerWeek * 52;
+
+export const mapOverviewViewport = {
```

Comment:
Consider Object.freezeing this so it doesn't accidentally get mutated in the course of the app being used.